### ISSUE TITLE
Could you share the model on Cityscape as well?

### ISSUE THREAD

**Issue by ChaoFang-TRI** [yanwei-li/PanopticFCN#4]
Thank you!

**Comment by yanwei-li** [comment]
Thanks for this suggestion. For the reason that some essential tools for Cityscapes (like panoptic dataset maker, instance-level random crop tools, etc.) have not been officially supported in Detectron2, we could not provide the training and inference code like a sub-project in Detectron2. We plan to release the full modified version and well as all the pretrained model later. Of course, it is easy to implement PanopticFCN for Cityscapes or other datasets with the currently released code.

**Comment by jinfagang** [comment]
@yanwei-li I am also interested on how panopticFCN preformance on cityscapes. It seems it's more convenient for cityscapes to train panoptic since it have mask png labeling already, however, detectron2 seems not have such dataloader out-of-box to use. That would be very nice if you can publish a cityscapes trained model!
